app/src/main/res/values/strings.xml

```diff
@@ -952,6 +952,34 @@
     <string name="hko_warning_text_tsunami" translatable="false">Tsunami Warning</string>
     <string name="hko_warning_text_fire_yellow_description" translatable="false">The Fire Danger Warning is Yellow and the fire risk is High.</string>
     <string name="hko_warning_text_fire_red_description" translatable="false">The Fire Danger Warning is Red and the fire risk is Extreme.</string>
+    <!-- Weather texts for MGM -->
+    <string name="mgm_weather_text_clear" translatable="false">@string/common_weather_text_clear_sky</string>
+    <string name="mgm_weather_text_slightly_cloudy" translatable="false">@string/common_weather_text_mainly_clear</string>
+    <string name="mgm_weather_text_partly_cloudy" translatable="false">@string/common_weather_text_partly_cloudy</string>
+    <string name="mgm_weather_text_mostly_cloudy" translatable="false">@string/common_weather_text_cloudy</string>
+    <string name="mgm_weather_text_rain_light" translatable="false">@string/common_weather_text_rain_light</string>
+    <string name="mgm_weather_text_rain" translatable="false">@string/common_weather_text_rain</string>
+    <string name="mgm_weather_text_rain_heavy" translatable="false">@string/common_weather_text_rain_heavy</string>
+    <string name="mgm_weather_text_sleet" translatable="false">@string/weather_kind_sleet</string>
+    <string name="mgm_weather_text_snow_light" translatable="false">@string/common_weather_text_snow_light</string>
+    <string name="mgm_weather_text_snow" translatable="false">@string/common_weather_text_snow</string>
+    <string name="mgm_weather_text_snow_heavy" translatable="false">@string/common_weather_text_snow_heavy</string>
+    <string name="mgm_weather_text_rain_showers_light" translatable="false">@string/common_weather_text_rain_showers_light</string>
+    <string name="mgm_weather_text_rain_showers" translatable="false">@string/common_weather_text_rain_showers</string>
+    <string name="mgm_weather_text_rain_showers_heavy" translatable="false">@string/common_weather_text_rain_showers_heavy</string>
+    <string name="mgm_weather_text_rain_showers_local">Local rain showers</string>
```

Comment:
Missing `translatable="false"`

Reply:
There are only a handful of MGM-specific weather terms, as opposed to HKO ones. Are you sure you want these marked as not translatable? (Fine with me either way. Just wanted to leave options open.)

Reply:
Alright, let's translate that one. I will continue the discussion for the other ones ;)

Reply:
Or should I just forget about the "local" part altogether and point to just `common_weather_text_rain_showers`?